Rocket.Chat/Managers/Model/SubscriptionManager.swift

```diff
@@ -339,6 +339,34 @@ extension SubscriptionManager {
             completion(response)
         }
     }
+
+    static func updateRoomDescription(subscription: Subscription, description: String, completion: @escaping MessageCompletion) {
+        let request = [
+            "msg": "method",
+            "method": "saveRoomSettings",
+            "id": "16",
+            "params": [subscription.rid, "roomDescription", description]
+        ] as [String: Any]
+
+        SocketManager.send(request) { response in
+            guard !response.isError() else { return Log.debug(response.result.string) }
+            completion(response)
+        }
+    }
+
+    static func updateRoomTopic(subscription: Subscription, topic: String, completion: @escaping MessageCompletion) {
+        let request = [
+            "msg": "method",
+            "method": "saveRoomSettings",
+            "id": "16",
```

Comment:
This param can be removed, the `id` is automatically added on `SocketManager.send()`